grammar.js

```diff
@@ -237,6 +240,60 @@ module.exports = grammar({
       ')'
     ),
 
+    ms_modifiers_declspec: $ => seq(
```

Comment:
Can you rename this to `ms_declaration_modifier` or `ms_declspec_modifier`?

I think that the singular word "modifier" makes sense because this node just represents *one* modifier, as opposed to a list of many.

Usually we avoid abbreviations like `decl` or `expr`, but I can see the argument that in this case the term "declspec" may be widely used. I'll leave the decision of "declspec_modifier" vs "declaration_modifier" up to you, I just wanted to throw out the suggestion of the non-abbreviated version.